src/resolvers.ts

```diff
@@ -14,8 +14,12 @@ export interface UserInput {
 export interface LoginInput {
   email: string;
   password: string;
+  rememberMe?: boolean | null;
 }
 
+export const TIME_EXPIRETION_DEFAULT = "1d";
```

Comment:
```suggestion
export const TIME_EXPIRATION_DEFAULT = "1d";
```
e já que ta fazendo assim, poderia deixar no env se quiser 🚀